src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java

```diff
@@ -103,6 +104,20 @@ public void createCase(@NotNull String caseType, @NotNull String dateReceived, @
         }
     }
 
+    public void migrateCase(@NotNull String caseType, @NotNull String dateReceived, @NotNull String fromCaseUUID) {
```

Comment:
Is caseType the source or target case type of the migration?

Reply:
It's the target case type